ui/app/(wrapped)/pilotage-reforme/page.tsx

```diff
@@ -0,0 +1,188 @@
+import { createParametrizedUrl } from "../../../utils/createParametrizedUrl";
+import { withAuth } from "../../../utils/security/withAuth";
+import { CartoSection } from "./stats/CartoSection";
+import { EvolutionIndicateursClesSection } from "./stats/EvolutionIndicateursClesSection";
+import { FiltersSection } from "./stats/FiltersSection";
+import { IndicateursClesSection } from "./stats/IndicateursClesSection";
+import { VueRegionAcademieSection } from "./stats/VueRegionAcademieSection";
+import {
+  Filters,
+  FiltersRegions,
+  Order,
+  PilotageReformeStatsQuery,
+  PilotageReformeStatsRegionsQuery,
+} from "./types";
+
+export default withAuth("pilotage_reforme/lecture", function PilotageReforme() {
+  const fetchPilotageReformeStats = async (query: PilotageReformeStatsQuery) =>
+    api.getPilotageReformeStats({ query }).call();
+
+  const fetchPilotageReformeStatsRegions = async (
+    query: PilotageReformeStatsRegionsQuery
+  ) => api.getPilotageReformeStatsRegions({ query }).call();
+
+  const router = useRouter();
+  const queryParams = useSearchParams();
+  const searchParams: {
+    filters?: Partial<Filters>;
+    order?: Partial<Order>;
+  } = qs.parse(queryParams.toString());
+
+  const filters = searchParams.filters ?? {};
+  const order = searchParams.order ?? { order: "asc" };
+
+  const setSearchParams = (params: {
+    filters?: typeof filters;
+    order?: typeof order;
+  }) => {
+    router.replace(
+      createParametrizedUrl(location.pathname, { ...searchParams, ...params })
+    );
+  };
+
+  const trackEvent = usePlausible();
+  const filterTracker = (filterName: keyof Filters) => () => {
+    trackEvent("pilotage-reforme:filtre", {
+      props: { filter_name: filterName },
+    });
+  };
+
+  const handleOrder = (column: Order["orderBy"]) => {
+    trackEvent("pilotage-reforme:ordre", { props: { colonne: column } });
+    if (order?.orderBy !== column) {
+      setSearchParams({ order: { order: "desc", orderBy: column } });
+      return;
+    }
+    setSearchParams({
+      ...filters,
+      order: {
+        order: order?.order === "asc" ? "desc" : "asc",
+        orderBy: column,
+      },
+    });
+  };
+
+  const handleFilters = (
+    type: keyof Filters | keyof FiltersRegions,
+    value: Filters[keyof Filters]
+  ) => {
+    setSearchParams({
+      filters: { ...filters, [type]: value },
+    });
+  };
+
+  const { data, isLoading: isLoading } = useQuery({
+    keepPreviousData: true,
+    staleTime: 10000000,
+    queryKey: ["pilotageReforme", filters],
+    queryFn: () => {
+      return fetchPilotageReformeStats({
+        ...filters,
+      });
+    },
+  });
+
+  const { data: dataRegions, isLoading: isLoadingRegions } = useQuery({
+    keepPreviousData: true,
+    staleTime: 10000000,
+    queryKey: ["pilotageReformeRegions", filters, order],
+    queryFn: () => {
+      return fetchPilotageReformeStatsRegions({
+        ...order,
+        ...filters,
+      });
+    },
```

Comment:
Une suggestion qui est pas moins lisible et rend le fichier plus concis selon moi, je te laisse choisir si tu préfères.
```suggestion
    queryFn: api.getPilotageReformeStatsRegions({         
        ...order,
        ...filters,
      }).call,
```

Reply:
Ça serait pas plutôt 

```
queryFn: api.getPilotageReformeStats({
      query: {
        ...filters,
      },
    }).call
```
?